examples/10_fs-router/src/entries.tsx

```diff
@@ -1,11 +0,0 @@
-import { fsRouter } from 'waku/router/server';
-
-declare global {
-  interface ImportMeta {
-    readonly glob: any;
-  }
-}
-
-export default fsRouter(import.meta.url, (file: string) =>
-  import.meta.glob('./pages/**/*.tsx')[`./pages/${file}`]?.(),
-);
```

Comment:
Actually, I want to keep this in the example. Can you revert changes in 10_fs-router?

Reply:
reverted 👍